src/scss/partials/_faq.scss

```diff
@@ -0,0 +1,37 @@
+.faq-box {
+  width: 50%;
+
+  > h3 {
+    margin: 30px 0 0 0;
+    font-weight: bold;
+    background-color: $text-color;
+    color: $background-color;
+    display: inline-block;
+    border-radius: 5px;
+
+    > a {
+      padding: 4px 13px;
+      display: inline-block;
+      position: relative;
+
+      &:hover:before {
+        display: block;
+      }
+
+      &:before {
+        padding-right: 13px;
+        position: absolute;
```

Comment:
Absolute positioning is used here to create an "anchor" icon `:before` FAQ title links.